umbral/pre.py

```diff
@@ -177,7 +180,8 @@ def verify(self) -> bool:
         s = self._bn_sig
         h = CurveBN.hash(e, v, params=self._umbral_params)
 
-        return s * g == v + (h * e)
+        result = s * g == v + (h * e)      # type: bool
```

Comment:
Isn't this redundant? We already said above that the result is `bool`.

Reply:
Yeah - perhaps redundant - Looking for precision on the charts, the return value type and the return type of the arithmetic operators are different.

Reply:
Different? I don't see why they're different.